src/screens/ShowcaseSheet.js

```diff
@@ -70,11 +70,11 @@ export default function ShowcaseScreen() {
 
   const { network } = useAccountSettings();
 
-  const {
-    data: { nfts: uniqueTokens },
-    isInitialLoading,
-  } = useLegacyNFTs({
-    address: accountAddress ?? '',
+  const userNftsStore = useUserNftsStore();
+  const uniqueTokens = userNftsStore(state => state.getData?.()?.nfts || []);
+  const isInitialLoading = userNftsStore(state => {
```

Comment:
there is `state.getStatus().isInitialLoading` you can use for `isInitialLoading`

also, I've just been combining like so:
```const { uniqueTokens, isInitialLoading } = userNftsStore(state => return { uniqueTokens: state.getData?.()?.nfts || [], isInitialLoading: state.getStatus().isInitialLoading });```

Reply:
Nice tip on the combo. Sorry, this is misleading on my end. I did want the behavior of `'loading'` state, but kept the isInitialLoading name to reduce the diff.

Reply:
If you create a new object in the selector, since the default equality function is `Object.is`, it's going to cause a re-render any time the store's state is updated. It's generally better not to merge different pieces of state into one object. If you need to, you can pass a different equality function (e.g. `shallow` from Zustand, `isEqual` from `react-native-fast-compare`) as the second arg, following the selector, so it can detect actual changes reliably.

For the state you need here, you should be able to use:

```ts
const isFetching = useUserAssetsStore(state => state.getStatus().isFetching);
```

I suppose `isFetching` and `isInitialLoading` could be made more consistent.